gcp-repo/metadata.p2.inf

```diff
@@ -10,7 +10,7 @@
 
 requires.1.namespace=org.eclipse.equinox.p2.iu
 requires.1.name=com.google.cloud.tools.eclipse.category
-requires.1.range=[1.0.0,1.2.0]
+requires.1.range=[1.0.0,1.3.0]
 requires.1.greedy=true
 
 units.1.id=com.google.cloud.tools.eclipse.category
```

Comment:
The line right below is `units.1.version=1.1.0`. Its value should have been 1.2.0.